assignments/function-conversion.js

```diff
@@ -4,27 +4,37 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
-
+let myFunction = () => {
+    console.log("Function was invoked!")
+};
+myFunction();
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
+let anotherFunction = (param) => param;
```

Comment:
I believe you already know this, but in functions with just 1 param you could skip the parenthesis for added cleanness.

Reply:
Aiit thanks